CHANGELOG.txt

```diff
@@ -2,6 +2,9 @@ master
 ======
 
 Changes:
+ - obspy.signal.spectral_estimation.PPSD:
+   * Added special handling option for infrasound data and global infrasound noise
+     models for plotting (see #2740)
```

Comment:
Please keep the changelog in alphabetical order. The only exception we usually make is to have `obspy.core` at the very top, but then everything else beneath in alphabetical order. I see another breach with the FDSN client log at the bottom, maybe while you're at it you can move that in the right spot too?

Reply:
Thanks - will do!